### DISCUSSION TITLE
external map location / add map menu item

### DISCUSSION THREAD

**Discussion by johnthebird** [devemux86/cruiser#62] [Q&A] [answered]
Hey there, it seems like storing and retrieving maps in/from external locations (eg SD card) is not supported any more. In earlier versions there was an add map menu item. Oruxmaps for instance can still be switched to external locations. Use case: I have a bunch of large mapsforge/openandro maps and want to share them across several apps. I do not want to download too often and use disk space as little as possible.
Anybody knows if this behavior is somehow still possible or is the feature planned to be reactivated?

Side note: 
Really like the performance and easy usage of cruiser. Built and checked the mapsforge android sample project and it is by far slower when zooming and moving within maps. Cruiser has the best performance I have seen so far regarding big offline .map files for outdoor navigation. Wonder about the coding delta ;-)

**Comment by Leeb64** [comment]
Settings/Application/Advanced/Menu/->Advanced
After that maps can be imported again

**Reply by johnthebird** [reply]
Thanks, that's it. Hidden but there

**Comment by devemux86** [comment, marked as answer]
Hi @johnthebird!

It is a bit "hidden" as there is an integrated offline maps downloader.

You can enable the menu "Open maps" and "Open map themes" in "Settings | Application | Advanced | Menu"
or search settings for "menu".

---

Cruiser can use 2 map engines: Mapsforge or VTM OpenGL).
We recommend the default VTM (OpenGL) for performance and extra features.

You can select it in "Settings | Map | Advanced | Rendering engine" (restart the app)
or search settings for "engine".

---

I am also the developer of [Mapsforge](https://github.com/mapsforge/mapsforge) and [VTM](https://github.com/mapsforge/vtm) map libraries.

**Reply by johnthebird** [reply]
Thanks a lot. Good to still have this feature. I will have a look into VTM if I find some time.
